frontend/src/components/common/wrappers/ModalWrapper.js

```diff
@@ -43,13 +45,6 @@ export class ModalWrapper extends Component {
     this.props.closeModal();
   };
 
-  renderTitle = () => (
```

Comment:
Does this modal no longer have a title?

Reply:
This block of code wasn't being used, I think it was there originally but wasn't removed when the component was refactored.